src/lib.rs

```diff
@@ -150,22 +149,6 @@ impl Hash {
     pub fn as_bytes(&self) -> &[u8; OUT_LEN] {
         &self.0
     }
-
-    /// The hexadecimal encoding of the `Hash`. The returned [`ArrayString`] is
-    /// a fixed size and doesn't allocate memory on the heap. Note that
-    /// [`ArrayString`] doesn't provide constant-time equality checking, so if
-    /// you need to compare hashes, prefer the `Hash` type.
-    ///
-    /// [`ArrayString`]: https://docs.rs/arrayvec/0.5.1/arrayvec/struct.ArrayString.html
-    pub fn to_hex(&self) -> ArrayString<[u8; 2 * OUT_LEN]> {
-        let mut s = ArrayString::new();
-        let table = b"0123456789abcdef";
-        for &b in self.0.iter() {
-            s.push(table[(b >> 4) as usize] as char);
-            s.push(table[(b & 0xf) as usize] as char);
-        }
-        s
-    }
```

Comment:
A slight breaking change. See the `Display` implementation below for the standard approach to write out bytes as hex in Rust. I can split into its own PR if preferred.